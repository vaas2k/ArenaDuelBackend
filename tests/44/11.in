1000 886317649
62213757 970080306 369952206 19237110 738591225 716694713 859856158 789550340 634931748 3886496 734606603 933891666 948615165 825692877 555440328 244149236 498644365 547837889 171451632 312105085 597397981 705661371 526111919 304462606 919298118 688210437 910093844 158330948 662804722 737101256 857741376 827713725 815221062 20912557 954008447 404422969 154022755 255067432 977588252 368981737 855448841 686354545 383410094 530727628 930546063 328070139 639950655 842392508 796752250 543712557 769675398 213041640 894636857 379728936 953524935 567919572 428963746 526479968 247165604 286005631 222461214 345359962 355222278 385415208 73604019 542956584 553464047 607387538 17972346 657674327 514618823 332081710 12286586 777457765 746416581 376716068 512333254 681538763 454181762 35307246 511384720 883722599 580672149 486890694 867214382 739097869 887473651 790360937 245691031 775003091 820912256 920163441 570038826 494716581 829513507 440261786 248562254 113811234 441662961 641362459 291424038 918071044 864739505 296526799 389699726 254676720 60385756 99234027 389032965 725162869 912358118 803501545 271290156 614128665 379053491 346704268 839863172 621041848 406422737 889064694 138650287 284181147 369389955 211352656 666823478 563946106 132425387 580252266 295606544 674077750 310499513 32939701 222862354 268675018 520451904 165548333 993541238 859137166 353535782 900353597 625285917 175604687 76555122 212388213 904557017 94806423 219097062 568873778 542398847 199654255 941165293 304766891 85989690 545755956 402905501 629922279 707100176 42997060 227499241 950666415 755726716 651368965 212251951 198080778 888760901 551314178 600483659 103702234 846604815 982531039 866887408 599342096 807162798 515388474 290579328 657687932 518771719 549268747 422647200 460158476 800948095 159927015 585746786 641387428 87511994 858754194 489807152 675917680 456606258 660939453 919972241 556446509 386570002 718460710 99649876 744613312 941197971 957412791 997873332 135585178 823082932 670481295 153950151 640400128 717413129 86208752 842915966 274294270 989328788 348114473 999906778 897857748 553302396 424629724 211126961 780982546 822499644 16971791 910814726 332498268 610120956 693037494 959744713 350455213 285913134 260746645 414765412 84204191 946224206 166993537 518037331 164758227 924898214 166236977 313253336 93763286 136343509 548294794 957541863 37056406 40999694 821471847 235299676 163343231 923700434 493447960 78948278 35160235 266646773 693162362 294319634 470529326 148567476 247142693 311611861 592572592 971063222 873044817 146386257 722680606 993588959 412607512 31333078 706429330 253646580 847741199 969961818 723028804 369076012 598031884 917234054 794029553 199145752 370611012 250076852 69062778 937112117 451057553 787173684 943450081 158621860 391058004 907865997 691827516 96717598 953286935 6422289 248254249 60905627 439686500 243008974 741122711 236275929 810704524 998895559 374039942 4185347 47956092 202737183 212678991 503631507 411870491 706997335 358863539 94922077 977282310 564484442 251276416 53759206 877771187 558066905 334100737 570928063 788837104 46795814 270366391 226339727 67037452 257904154 994967761 202692800 580904736 921472354 836029113 346057178 83529297 279963487 909396485 26019733 290572359 360914230 851049231 960800697 463477904 682347673 922954444 921022151 660197993 651031624 289310491 24007265 443027185 671804829 84090382 305733858 46654261 382688402 723362454 549488676 594328607 949655007 510159848 959966694 187715163 836805444 102711082 469205676 295855656 907351302 817079020 491946187 73595593 62728841 444552193 535101071 909152493 382392341 666056987 635691062 289337606 109244100 583583025 915683699 299570334 358743998 383381276 113375691 133860833 639227591 841873182 367690197 740605129 18622069 341709321 24947721 855471299 883450972 526969988 235231107 229786323 68985951 352399911 286421292 345333653 759495535 696854121 835124042 492228634 651790868 616451078 171414748 342288765 987958410 692065314 659381604 672514075 764714679 391615501 42888340 350860668 742161110 225893501 555226854 850743565 518848251 704383164 551442463 328316719 287250317 888852739 351773405 199687056 432475955 696574773 313796920 512910497 14762372 387968759 355160592 760559628 340949994 644386639 88351946 830838504 882098475 546721411 822140120 992566523 817490268 86549023 190232709 991627410 962190785 165800934 19937327 857934458 350775802 262835778 701253939 843114603 185223840 569498622 64978758 576826766 728799808 199424584 609802138 865938009 311330982 423561231 379463646 948680968 178704156 821139526 177387607 830214063 379239382 64980910 860037442 611369086 283880102 746171646 885069950 614297012 10840748 192278467 172674371 149783027 671506628 292966933 642736856 938274357 471507893 698554568 886160082 679933630 569211809 48359079 199378062 452052340 77945456 913739267 75733802 343140610 362859913 433043420 913772111 267470025 913837110 510917150 738544322 627486152 408554546 786211379 801438583 381661513 740924145 530351461 76683406 989534489 348174827 91136712 72419446 725385413 545385071 645680089 931528757 619165531 478562921 518391273 419973749 431379917 379338977 584218113 296519682 484517695 891541791 261325024 449655755 997775558 925925617 246835298 870967844 464053643 683771344 995729234 663450751 490446831 752921038 376001329 994676627 853131374 915414094 401736496 898975206 605079225 579439022 756247790 268881482 500918649 757715616 673914444 558925571 131884804 563878162 296247349 395730054 215845074 742621942 498391860 451910951 45648318 977435412 169805660 75980531 707459007 609826277 221480894 658928190 19648405 272573058 18817679 199078584 723989882 352841606 645187625 573844088 684731315 41463655 572162017 797495926 867794760 973248323 223376025 358050055 684249784 631535749 632738469 250540595 736071372 696722333 821281310 22306683 831921011 510578002 634218152 875515561 533916190 986061109 398714725 357929010 53514460 270738303 706015392 743209198 307170589 889416056 753298118 16383835 903612672 457109001 141393693 466267866 925815737 435872105 186177081 335936708 971498345 524072608 648906379 191431708 203392591 495308590 219994224 166738774 730720375 253805472 466670445 144352789 60220354 795315491 897429470 581076722 849704398 132463059 964522168 940345315 107913675 617183837 152862767 576491252 965933918 170235277 949904210 5327827 207620702 342687915 275298211 603349536 562731928 873948287 463131905 598201694 824090265 932793134 236126305 915363148 266233123 782383743 964000223 327791206 246131997 877588779 438361473 737186964 137753189 943128236 422504106 358585132 288975973 784825351 158102083 521375027 385450962 182700580 151962591 694462053 462517641 981223658 738499048 8557280 216843223 654930234 216968553 302218367 652667632 382076028 89348393 640114810 663726941 160341642 946523075 247486229 431171573 572763979 53744171 242855963 916446588 777291674 398733855 497730160 496048757 459731183 8049462 395166809 772607499 47035515 326757843 757169156 570026461 779047456 150941988 804451900 250699430 201669614 369911590 432227909 847020471 816713052 708747739 416434609 791303799 260236807 823959994 694399725 704522287 251061228 669396290 765225437 193604369 712458331 223010643 432837079 81449171 189548980 766283618 532215075 268557259 92413455 430705645 122038876 626828487 227077003 781249985 621321186 682056376 429121838 21326920 122416521 234536588 475463378 150699930 272317948 241758693 855719828 670625664 882583448 823712655 271033057 133659133 707778790 437305398 105884380 864830503 517809561 190653793 557456954 443024000 353556899 323630811 594296220 474430872 974884826 2716858 952300568 595608694 128318188 436362851 838458113 820155054 98728881 247188776 961157860 738461969 682188272 443857881 872203466 172323266 958146571 341357533 430016573 263346707 113784714 258974516 125992122 594834131 294254976 551549071 235014879 866852107 241950981 456051877 589113727 52897727 220101945 925946261 65254673 592506473 481222529 852565904 834662630 893646994 54686937 607626226 266443225 241686927 825433069 438429816 724248774 933059149 867891625 620049872 134503993 472709651 653074579 902006744 286379239 136391099 155144768 366482447 665986843 255078227 630961747 351302371 736126201 956509072 369004083 475333939 610961873 197961820 487863447 146417552 308709329 630955887 301968476 321845347 521867413 230032515 832546453 181139957 776453233 852319660 232028767 189333022 499974994 50516296 668403867 292022871 525724415 337085741 176307643 506333731 69472541 662270572 25410005 510366836 689801325 423816750 790513425 649288855 886098340 588636260 650584352 732559581 13608656 400596945 317418464 204229345 679016349 500526831 734248695 731259012 612551645 639436552 742489900 592523018 426367492 100326880 684356756 191843641 474893938 331646433 289978577 641486411 488556695 24007667 301698254 353492824 927187323 562896741 367364623 66087107 214331513 252681149 114250323 738216112 371358242 821910324 27623139 579598033 92479503 133939391 86526848 445437479 672887557 260007029 625744819 333149493 484796388 753175962 150167780 756398810 404998204 699417837 94416981 453808857 850602005 20026879 63809538 603919336 527380961 858103874 498188833 757361206 443225927 458665718 216681076 998562698 164781745 244083740 929688334 155731267 640505930 595544226 449658669 615541100 153428263 643877637 308122498 449331473 942098366 212340583 15398916 278279528 316106217 592129462 327207064 728638276 159484868 833896469 787529665 340563074 300742528 856977057 582905246 161875721 114722798 496940725 419769019 265317334 38646942 305428189 7450915 409693462 177323779 298912795 636475153 487128655 547811410 820941627 96324803 92247192 349846015 484402849 549359931 334606245 302932646 680929542 822844113 38325874 251703122 157430024 594365366 145341550 650558564 970936452 133072945 237491016 378346142 672853205
