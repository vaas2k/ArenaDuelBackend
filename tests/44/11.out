568 329 606 892
